explore/src/main/scala/explore/common/GroupQueries.scala

```diff
@@ -54,10 +54,13 @@ object GroupQueries:
       )
       .void
 
-  def createGroup[F[_]: Async](programId: Program.Id)(using
+  def createGroup[F[_]: Async](programId: Program.Id, parentId: Option[Group.Id])(using
     FetchClient[F, ObservationDB]
   ): F[Grouping] =
     CreateGroupMutation[F]
       .execute:
-        CreateGroupInput(programId = programId.assign)
+        CreateGroupInput(
```

Comment:
Is this undoable?

Reply:
No since removing groups is not supported yet